conf/messages

```diff
@@ -566,8 +566,8 @@ page.iht.application.overview.common.estatevalue = Based on the details you have
 page.iht.application.overview.common.helptext.part1 = If you find any more assets owned by the deceased that weren’t included in this estate report, let HMRC know by using a
 page.iht.application.overview.common.helptext.part2 = Only do this if these assets increase the value of the estate over the Inheritance Tax threshold.
 page.iht.application.overview.common.return = Return to your Inheritance Tax estate reports
-page.iht.application.overview.common.viewcopy = View a copy of this estate report
-page.iht.application.overview.common.viewcertificate = View clearance certificate
+page.iht.application.overview.common.viewcopy = View a copy of this estate report (PDF, &lt;500KB)
```

Comment:
We shouldn't be storing HTML in the content file. This should be stored in the individual view files.